sympy/solvers/pde.py

```diff
@@ -662,26 +677,26 @@ def pde_1st_linear_constant_coeff(eq, func, order, match, solvefun):
 def pde_1st_linear_variable_coeff(eq, func, order, match, solvefun):
     r"""
     Solves a first order linear partial differential equation
-    with variable coefficients. The general form of this partial differential equation is
+    with variable coefficients. The general form of this partial
+    differential equation is
 
     .. math:: a(x, y) \frac{df(x, y)}{dx} + a(x, y) \frac{df(x, y)}{dy}
-                + c(x, y) f(x, y) - G(x, y)
+                + c(x, y) f(x, y) = G(x, y)
 
-    where `a(x, y)`, `b(x, y)`, `c(x, y)` and `G(x, y)` are arbitrary functions
-    in `x` and `y`. This PDE is converted into an ODE by making the following transformation.
+    where `a(x, y)`, `b(x, y)`, `c(x, y)` and `G(x, y)` are arbitrary
+    functions in `x` and `y`. This PDE is converted into an ODE by
+    making the following transformation.
 
-    1] `\xi` as `x`
+    1. `\xi` as `x`
 
-    2] `\eta` as the constant in the solution to the differential equation
-    `\frac{dy}{dx} = -\frac{b}{a}`
+    2. `\eta` as the constant in the solution to the differential
+       equation `\frac{dy}{dx} = -\frac{b}{a}`
 
     Making the following substitutions reduces it to the linear ODE
 
     .. math:: a(\xi, \eta)\frac{du}{d\xi} + c(\xi, \eta)u - d(\xi, \eta) = 0
 
-    which can be solved using dsolve.
-
-    The general form of this PDE is::
+    which can be solved using ``dsolve``.
```

Comment:
dsolve -> pdsolve

Reply:
This one is correct, though. It says that after making the substitutions it can be solve using ``dsolve`` that is what is described in the book and the code is doing.